HelpSource/editor.js

```diff
@@ -129,7 +129,13 @@ const selectRegion = (options = { flash: true }) => {
     }
 }
 
-/* returns the code selection or line */
+const evalLine = () => {
+    // Call to C++ help browser. `evaluateSelection(false)` evaluates a line. It will execute
```

Comment:
maybe a little verbose comment, perhaps something like "call c++ evaluation function in order to capture shift-enter" would be fine. the comment should go on top of the function signature.